macro-analytics/README.md

```diff
@@ -0,0 +1,18 @@
+
+# Build a Google Analytics dashboard 🖥️ with an Apps Script, a Sheets add-on,and Google Sites
+
+This is an Apps Script sample to automate the setup and summary of charts
+from Google Analytics into Google Sheets. Charts can optionally be inserted
+into a new Google Site and be scheduled to run on a routine basis to share data
+in professional dashboard outside of a Google spreadsheet for presentation
+purposes.
+
+What you will need:
+
+1. A Google Analytics account that has been collecting data from a website.
+1. A Google Sheet created by the same Google account that has access to your
+   Google Analytics account.
+
+For full instructions with screenshots please visit [this blog post.](https://medium.com/@TechandEco/apps-script-macro-to-automate-daily-google-analytics-reports-%EF%B8%8F-in-a-google-sheet-668263ab6c8c)
```

Comment:
It would be useful to at least mention that this code was generated from a macro recorder and a link on how to do that, and then mention your blog post for more information

Reply:
good call, added!